sami2py/_core_class.py

```diff
@@ -93,26 +90,23 @@ def __init__(self, tag, lon, year, day, outn=False, test=False):
 
     def __repr__(self):
         """Make a printable representation of a Model object
-
         Returns
         -------
         out : (string)
             string containing a printable representation of a Model object
-
         Examples
         --------
         Load the model
             ModelRun = sami2py.Model(tag='run_name', lon=0, year=2012, day=210)
         Check the model information
             ModelRun
-
         """
 
         out = ['']
         out.append('Model Run Name = ' + self.tag)
         out.append(('Day {day:03d}, {year:4d}').format(day=self.day,
                                                        year=self.year))
-        out.append(('Longitude = {lon:d} deg').format(lon=self.lon0))
+        out.append(('Longitude = {lon:5.1f} deg').format(lon=self.lon0))
```

Comment:
```suggestion
        out.append(('Longitude = {:5.1f} deg').format(self.lon0))
```

Since you aren't really using the named functionality of format.

Reply:
Also elsewhere.  But it's fine to keep it if you like this style.